### PR TITLE
Update telemetry.md

### PR DESCRIPTION
**Pull request by lilyjazz**
update telemetry information about TiDB.

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

update telemetry information about TiDB in v5.0.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: telemetry.md [hunk 1]
```diff
@@ -18,10 +18,11 @@ TiDB、TiUP 及 TiDB Dashboard 默认会收集使用情况信息，并将这些
 
 ### TiDB
 
-当 TiDB 遥测功能开启时，TiDB 集群将会以 24 小时为周期收集使用情况信息并分享给 PingCAP，包括（但不限于）：
+当 TiDB 遥测功能开启时，TiDB 集群将会以 6 小时为周期收集使用情况信息并分享给 PingCAP，包括（但不限于）：
 
 - 随机生成的遥测标示符
 - 集群的部署情况，包括各个组件所在的硬件信息（CPU、内存、磁盘）、组件版本号、操作系统版本号等
+- 系统的组件使用情况，包括 Async Commit 是否实际被使用等
```
Comment (qiancai) [inline telemetry.md:25]:
```suggestion
- 系统的组件使用情况，包括 Async Commit 是否被使用等
```

File: telemetry.md [hunk 2]
```diff
@@ -18,10 +18,11 @@ TiDB、TiUP 及 TiDB Dashboard 默认会收集使用情况信息，并将这些
 
 ### TiDB
 
-当 TiDB 遥测功能开启时，TiDB 集群将会以 24 小时为周期收集使用情况信息并分享给 PingCAP，包括（但不限于）：
+当 TiDB 遥测功能开启时，TiDB 集群将会以 6 小时为周期收集使用情况信息并分享给 PingCAP，包括（但不限于）：
 
 - 随机生成的遥测标示符
 - 集群的部署情况，包括各个组件所在的硬件信息（CPU、内存、磁盘）、组件版本号、操作系统版本号等
+- 系统组件的使用情况，包括 Async Commit 是否被使用等
```
Comment (qiancai) [inline telemetry.md:25]:
```suggestion
- 系统组件的使用情况，例如 Async Commit 是否有被使用
```

File: telemetry.md [hunk 3]
```diff
@@ -18,10 +18,11 @@ TiDB、TiUP 及 TiDB Dashboard 默认会收集使用情况信息，并将这些
 
 ### TiDB
 
-当 TiDB 遥测功能开启时，TiDB 集群将会以 24 小时为周期收集使用情况信息并分享给 PingCAP，包括（但不限于）：
+当 TiDB 遥测功能开启时，TiDB 集群将会以 6 小时为周期收集使用情况信息并分享给 PingCAP，包括（但不限于）：
 
 - 随机生成的遥测标示符
 - 集群的部署情况，包括各个组件所在的硬件信息（CPU、内存、磁盘）、组件版本号、操作系统版本号等
+- 系统组件的使用情况，例如 Async Commit 是否有被使用
```
Comment (breeswish) [inline telemetry.md:25]:
```suggestion
- 系统组件的使用情况，例如 Async Commit 功能是否有被使用
```

Comment (lilyjazz) [reply telemetry.md:25]: OK。

**Review by breeswish — APPROVED** [review]

**Pull request merged**